server/internal/app/graphql.go

```diff
@@ -82,3 +89,68 @@ func GraphqlAPI(conf config.GraphQLConfig, dev bool) echo.HandlerFunc {
 		return nil
 	}
 }
+
+// customErrorPresenter handles custom GraphQL error presentation by converting various error types
+// into localized GraphQL errors.
+//
+// Parameters:
+//   - ctx: The context containing language preferences
+//   - e: The error to be presented
+//   - devMode: If true, includes additional debugging information
+//
+// Returns a GraphQL error with localized messages and optional debug information.
```

Comment:
We don't need long comments for functions, the implementation is already understandable.